neqo-http3/src/features/extended_connect/webtransport_streams.rs

```diff
@@ -23,7 +23,7 @@ pub struct WebTransportRecvStream {
     stream_id: StreamId,
     stream_info: Http3StreamInfo,
     events: Box<dyn RecvStreamEvents>,
-    session: Rc<RefCell<WebTransportSession>>,
+    session: Rc<RefCell<webtransport_session::Session>>,
```

Comment:
```suggestion
    session: Rc<RefCell<webtransport::Session>>,
```
I'd like it looking like this, but it may mean too much refactoring :-)

Reply:
I too prefer `webtransport::Session` or `webtransport::session::Session`. I will follow up with another pull request.